src/stores/CustomRoomTagStore.js

```diff
@@ -17,6 +17,7 @@ import dis from '../dispatcher';
 import * as RoomNotifs from '../RoomNotifs';
 import RoomListStore from './RoomListStore';
 import EventEmitter from 'events';
+import * as _ from "lodash";
```

Comment:
I think we ave previously imported individual functions from lodash in order to try not to pull in the whole thing (although I suspect webpack 4's tree shaking will remove the rest anyway nowadays, so it probably doesn't make much difference.

Reply:
> I think we have previously imported individual functions from lodash in order to try not to pull in the whole thing (although I suspect webpack 4's tree shaking will remove the rest anyway nowadays, so it probably doesn't make much difference.

Ok, changed it anyways :+1: